include/swift/AST/GenericSignature.h

```diff
@@ -223,6 +223,17 @@ class alignas(1 << TypeAlignInBits) GenericSignature final
       });
   }
 
+  /// Compute the number of conformance requirements in this signature.
+  unsigned getNumConformanceRequirements() const {
+    unsigned result = 0;
```

Comment:
This could be

```cpp
return llvm::count_if(getRequirements(),
    [&] (const Requirement &req) { return req.getKind() == RequirementKind::Conformance }));
```

although that's not entirely better.